### DISCUSSION TITLE
Direct access to gallery from Native Camera?

### DISCUSSION THREAD

**Discussion by romifauzi** [yasirkula/UnityNativeCamera#87] [Q&A] [answered]
Thank you so much for this great addons @yasirkula! I have questions, is it possible to show direct access to the Gallery in the NativeCamera? similar to the Camera native app on each platform, where the user can tap the last image/video taken to go to the gallery...

**Comment by yasirkula** [comment, marked as answer]
Thank you! Any customizations to the native camera UI apart from the provided API is, AFAIK, not possible. So I'd recommend you to create a UI in Unity which lets user pick NativeCamera or NativeGallery.

**Reply by romifauzi** [reply]
Hi, Thank you so much for the fast response, yeah I was thinking that too as an alternative, another quick question (although I'm doubting about this), but is there a way to show unity button on top of this NativeCamera ui?

**Reply by yasirkula** [reply]
Unfortunately not possible with this plugin :D To do that, the camera feed must be rendered inside Unity but NativeCamera doesn't work like that.

**Reply by romifauzi** [reply]
I see, okay then. Thank you so much for confirming this! All the best!
